### PR TITLE
Add columns to taxonomy screens

### PR DESCRIPTION
**Pull request by contactashish13**
https://github.com/semperfiwebdesign/aioseop-pro/issues/34

### CODE REVIEW

**Comment by wpsmort** [comment]
@contactashish13 I get the following fatal error upon activation:

Fatal error: Uncaught BadMethodCallException: Method add_help_text_links doesn't exist in all-in-one-seo-pack-pro/admin/aioseop_module_class.php on line 54

I am taking the code from this PR and merging it with the master branch of Pro by adding the /pro/ directory and changing define( 'AIOSEOPPRO', false ); to true.  

Let me know if you need any more information.

**Comment by contactashish13** [comment]
@wpsmort my changes are all in the free plugin and have nothing to do with the pro directory. I see that the file `admin/aioseop_module_class.php` in PRO contains the method `add_help_text_links` but the free one does not. I'm not sure why so there is obviously something out of sync between the two. This error is unrelated to my changes.

**Comment by wpsmort** [comment]
@contactashish13 I understand your comments, however, the fatal error only occurs when I add the /pro/ directory from the master branch and change the define('AIOSEOPPRO') to true.  It does not happen when I test the master branch on it's own or when I test your PR code.

**Comment by michaeltorbert** [comment]
@contactashish13 
The method is here: https://github.com/semperfiwebdesign/all-in-one-seo-pack/blob/53b0a5f9c74941979d645caa4866145287f7e944/admin/aioseop_module_class.php#L1698

They aren't "out of sync" other than Pro doesn't have any of the 3.0 changes from free.

@wpsmort I've created Pro with this PR and have uploaded it to slack. I tested to make sure it didn't have a fatal error.

**Comment by contactashish13** [comment]
To resolve the fatal error:
- Ben needs to resolve https://github.com/semperfiwebdesign/aioseop-pro/issues/473
- that PR needs to be merged to master branch of PRO
- this PR (https://github.com/semperfiwebdesign/all-in-one-seo-pack/pull/2304) should be tested again after that merge

**Comment by michaeltorbert** [comment]
@contactashish13 Why does that PR need to go into Pro's master branch instead of its 3.0 branch?

**Comment by contactashish13** [comment]
Because Steve was testing with the master branch but I suppose it makes more sense to go into the 3.0 branch and he should test it from there.

**Comment by wpsmort** [comment]
This is tested and is good.

File: aioseop_class.php [hunk 1]
```diff
@@ -833,10 +833,18 @@ function __construct() {
 			add_action( 'all_admin_notices', array( $this, 'visibility_warning' ) );
 			add_action( 'all_admin_notices', array( $this, 'woo_upgrade_notice' ) );
 
+			if ( ! AIOSEOPPRO ) {
+				// add_action('all_admin_notices', array( $this, 'woo_upgrade_notice'));
```
Comment (EkoJR) [inline aioseop_class.php]: Unsure why this is being added.

File: aioseop_class.php [hunk 2]
```diff
@@ -833,10 +833,18 @@ function __construct() {
...
+			}
+
+			if ( AIOSEOPPRO ) {
+				add_action( 'registered_taxonomy', 'aioseop_taxonomy_post_register', 10, 1 );
```
Comment (EkoJR) [inline aioseop_class.php]: If this is for Pro only, would it be better in one of the Pro files?

File: all_in_one_seo_pack.php [hunk 3]
```diff
@@ -417,6 +417,10 @@ function aioseop_init_class() {
 		require_once( AIOSEOP_PLUGIN_DIR . 'admin/display/menu.php' );
 		require_once( AIOSEOP_PLUGIN_DIR . 'admin/class-aioseop-notices.php' );
 
+		if ( AIOSEOPPRO ) {
+			require_once( AIOSEOP_PLUGIN_DIR . 'inc/aioseop_taxonomy_functions.php' );
```
Comment (EkoJR) [inline all_in_one_seo_pack.php]: If this is for Pro only, would this be better in `pro/class-aio-pro-init.php`?

File: aioseop_class.php (hunk 2 above)
Comment (contactashish13) [reply aioseop_class.php]: @EkoJR I agree. PR https://github.com/semperfiwebdesign/aioseop-pro/pull/481 replaces this one

File: all_in_one_seo_pack.php (hunk 3 above)
Comment (contactashish13) [reply all_in_one_seo_pack.php]: as above

**Comment by contactashish13** [comment]
@wpsmort PR https://github.com/semperfiwebdesign/aioseop-pro/pull/481 replaces this one.

**Comment by wpsmort** [comment]
Closing this PR as it is replaced by https://github.com/semperfiwebdesign/aioseop-pro/pull/481

**Pull request closed without merging**